test/e2e/localnet-underlay.go

```diff
@@ -161,3 +163,46 @@ func bridgeMapping(physnet, ovsBridge string) BridgeMapping {
 		ovsBridge: ovsBridge,
 	}
 }
+
+func createVLANInterface(deviceName string, vlanID string, ipAddress *string) error {
```

Comment:
maybe it wouldn't hurt to have this function idempotent ... 

It's really not needed right now, since we only use it from a single place, but could be helpful going forward.

Reply:
Done.